packages/webapp/components/posts/PostWidgets.tsx

```diff
@@ -49,13 +36,10 @@ export function PostWidgets({ postById }: PostWidgetsProps): ReactElement {
 
   return (
     <PageSidebar>
-      {postById && <ShareBar post={postById.post} />}
+      <ShareBar post={post} />
       <ShareMobile share={sharePost} />
-      {postById?.post && tokenRefreshed && (
-        <FurtherReading
-          currentPost={postById.post}
-          className="laptopL:w-[19.5rem]"
-        />
+      {tokenRefreshed && (
```

Comment:
The check for `postById` was there on purpose because it's being fetched on load.
If this is not defined this might cause an error.

```suggestion
      {post && tokenRefreshed && (
```